server/manager/src/main/java/org/apache/accumulo/manager/tableOps/bulkVer2/PrepBulkImport.java

```diff
@@ -183,8 +183,15 @@ private void checkForMerge(final long tid, final Manager manager) throws Excepti
     VolumeManager fs = manager.getVolumeManager();
     final Path bulkDir = new Path(bulkInfo.sourceDir);
 
-    int maxTablets = Integer.parseInt(manager.getContext().getTableConfiguration(bulkInfo.tableId)
-        .get(Property.TABLE_BULK_MAX_TABLETS));
+    String value = manager.getContext().getTableConfiguration(bulkInfo.tableId)
+        .get(Property.TABLE_BULK_MAX_TABLETS);
+    if (value == null) {
+      value = Property.TABLE_BULK_MAX_TABLETS.getDefaultValue();
+      log.info("Property not found " + Property.TABLE_BULK_MAX_TABLETS + " using default: " + value
+          + " for tableId: " + bulkInfo.tableId + " using default: " + value);
+    }
+
+    int maxTablets = Integer.parseInt(value);
```

Comment:
This is a change in behavior that causes it to use a different value than the user intended to specify when there is a typo, instead of erroring out. It also does so with merely an info message, and not even a warning about the error. This seems completely unrelated to this PR and could be evaluated on its own in a different PR.

Reply:
Fixed it e9588a17cc - the issue was introduced because findbug spotted the value could be null - so Integer.parseInt was not advised.  Modified it to throw IllegalStateException if the value could not be found instead of continuing.

Reply:
Do you mean spotbugs? We don't use findbugs anymore. I don't see it failing in our current main branch. Your changes to configuration shouldn't have affected whether null was possible to be returned here or not. I actually don't think it's actually possible for it to return null in this case. It has a default value that would be returned if it were null. Wrapping it with requiresNonNull would probably suffice to explicitly throw NPE and not trigger spotbugs, though.

Reply:
With the original code unchanged the maven build / verify fails with the following:

```
[ERROR] Medium: Possible null pointer dereference in org.apache.accumulo.manager.tableOps.bulkVer2.PrepBulkImport.checkForMerge(long, Manager) due to return value of called method [org.apache.accumulo.manager.tableOps.bulkVer2.PrepBulkImport, org.apache.accumulo.manager.tableOps.bulkVer2.PrepBulkImport] Method invoked at PrepBulkImport.java:[line 186]Known null at PrepBulkImport.java:[line 187] NP_NULL_ON_SOME_PATH_FROM_RETURN_VALUE
```

Wrapping value in requireNonNull fails with the same message.

```
  requireNonNull(value,"The property: " + Property.TABLE_BULK_MAX_TABLETS.getKey()
          + " was not found for tableId: " + bulkInfo.tableId);
```